Makefile

```diff
@@ -0,0 +1,34 @@
+test:
+	poetry run pytest -v
```

Comment:
how do you feel about adding a make command that checks for and installs poetry before you use it?

Reply:
Or possibly just adding a getting started/readme that explains the process of getting up and running ....

Reply:
Just added a [README.md](https://github.com/omnivector-solutions/jobbergate-api-fastapi/blob/dev/README.md)